packages/eslint-plugin/docs/rules/TEMPLATE.md

```diff
@@ -1,6 +1,10 @@
+---
+description: '<Description from rule metadata here>'
+---
+
 > 🛑 This file is source code, not the primary documentation location! 🛑
 >
-> See **https://typescript-eslint.io/rules/your-rule-name** for documentation.
+> See **https://typescript-eslint.io/rules/RULE_NAME_REPLACEME** for documentation.
```

Comment:
unrelated?

Reply:
it was just me fleshing out the template
i noticed this was super easy to miss this so I made it more obvious!

Reply:
I like it :)